facade/host.go

```diff
@@ -301,33 +307,43 @@ func (f *Facade) GetHostKey(ctx datastore.Context, hostID string) ([]byte, error
 func (f *Facade) ResetHostKey(ctx datastore.Context, hostID string) ([]byte, error) {
 	defer ctx.Metrics().Stop(ctx.Metrics().Start("Facade.ResetHostKey"))
 	glog.V(2).Infof("Facade.ResetHostKey: id=%s", hostID)
+	alog := f.auditLogger.Message(ctx, "Resetting Host Key").
+		Action(audit.Update).ID(hostID).Type(host.GetType())
 
 	var value host.Host
 	if err := f.hostStore.Get(ctx, host.HostKey(hostID), &value); err != nil {
-		return nil, err
+		return nil, alog.Error(err)
 	}
-	return f.generateDelegateKey(ctx, &value)
+	key, err := f.generateDelegateKey(ctx, &value)
+	return key, alog.Error(err)
 }
 
 // RegisterHost attempts to register a host's keys over ssh, or locally if it's
 // the current host.
 func (f *Facade) RegisterHostKeys(ctx datastore.Context, entity *host.Host, nat utils.URL, keys []byte, prompt bool) error {
 	defer ctx.Metrics().Stop(ctx.Metrics().Start("Facade.RegisterHostKeys"))
-	return auth.RegisterRemoteHost(entity.ID, nat, entity.IPAddr, keys, prompt)
+	alog := f.auditLogger.Message(ctx, "Registering Host Keys").Entity(entity).Action(audit.Update)
+	return alog.Error(auth.RegisterRemoteHost(entity.ID, nat, entity.IPAddr, keys, prompt))
 }
 
 // SetHostExpiration sets a host's auth token
 // expiration time in the HostExpirationRegistry
 func (f *Facade) SetHostExpiration(ctx datastore.Context, hostid string, expiration int64) {
 	defer ctx.Metrics().Stop(ctx.Metrics().Start("Facade.SetHostExpiration"))
+	alog := f.auditLogger.Message(ctx, "Setting Host Expiration Registry").
+		Action(audit.Update).ID(hostid).Type(host.GetType())
```

Comment:
Should probably log `expiration`. Not sure what the units are, but ideally it should be recorded in some human-readable format.